app/controllers/v1/sessions_controller.rb

```diff
@@ -306,8 +306,9 @@ def login_stats(status, error = nil)
       when :success
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY, tags: [VERSION_TAG]) if type == 'signup'
         StatsD.increment(STATSD_LOGIN_STATUS_SUCCESS, tags:)
-        Rails.logger.info("LOGIN_STATUS_SUCCESS, tags: #{tags}")
-        Rails.logger.info("SessionsController version:v1 login complete, user_uuid=#{@current_user&.uuid}")
+        log_tags = tags.dup << "icn:#{@current_user.icn}"
+        Rails.logger.info("LOGIN_STATUS_SUCCESS, tags: #{log_tags}")
```

Comment:
can you just add `icn` to the log itself instead of duplicating the tags? (or could just define the log without using tags since the tags actually seem like they're there for the statsd)